index.js

```diff
@@ -74,7 +77,7 @@ function createReloadable (app, require) {
           currentResponse = null
           currentNext = null
         } else {
-          process.abort()
+          throw error
```

Comment:
👍

Reply:
I was seriously worried that with those C++ node errors we had sunk the boat

Reply:
Yeah those errors looked worrisome, glad it wasn't so bad to fix

Reply:
Nice and much better indeed 👌